CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## 5.0.0
+
+**Note:** This release is based off of 5.0.0-rc3. These are the changes compared to 5.0.0-rc3:
+
+- Removed deprecated `addAttributionData` functions and `AttributionNetwork` enum
+    https://github.com/RevenueCat/purchases-android/pull/506
+- Added retries with exponential backoff when reconnecting to Google's billing service
+    https://github.com/RevenueCat/purchases-android/pull/489
+- Updated to BillingClient 4.1
+    https://github.com/RevenueCat/purchases-android/pull/500
+- Prevent Amazon from swallowing exceptions
+    https://github.com/RevenueCat/purchases-android/pull/499
+- Improvements on Amazon implementation preventing a lot of issues related to price and currency parsing
+    https://github.com/RevenueCat/purchases-android/pull/503
+
```

Comment:
I feel like this section should be at the bottom, assuming that most people seeing this would be coming from 4.x.x